src/js/actions/superselect.js

```diff
@@ -622,8 +622,12 @@ define(function (require, exports) {
                         }
 
                         var artboardNested = false;
+                        var moveHandled  = false;
```

Comment:
These two flags seem redundant. If `artboardNested` is true, we `getLayerOrder` and if `moveHandled` is true, we `getLayerOrder` ?

There has to be a cleaner way of doing this, or at least a comment explaining the need for two flags.